style.css

```diff
@@ -7,24 +7,23 @@ body {
 }
 
 nav {
-  position: ;
+  position: absolute;
```

Comment:
- [ ] The position absolute is taking you nav out of the flow of your webpage. Hence, making objects to get over each other. You can either change the nav position back to its default `block` or you can add some margin-top to the elements that are under it to move them further down and avoid the superposition.

![image](https://user-images.githubusercontent.com/79114233/138214830-e5795655-478b-4b3b-9656-80ca42d95e67.png)
 _amount of space that needs to be between the top of your page and your headline elements_